apps/trivium/benches/kreyvium_bool.rs

```diff
@@ -0,0 +1,77 @@
+use tfhe::prelude::*;
+use tfhe::{generate_keys, set_server_key, unset_server_key, ConfigBuilder, FheBool};
+
+use tfhe_trivium::KreyviumStream;
+
+use criterion::Criterion;
+
+pub fn kreyvium_bool_gen(c: &mut Criterion) {
+    let config = ConfigBuilder::all_disabled().enable_default_bool().build();
+    let (client_key, server_key) = generate_keys(config);
+
+    let key_string = "0053A6F94C9FF24598EB000000000000".to_string();
+    let mut key = [false; 128];
+
+    for i in (0..key_string.len()).step_by(2) {
+        let mut val: u8 = u8::from_str_radix(&key_string[i..i + 2], 16).unwrap();
+        for j in 0..8 {
+            key[8 * (i >> 1) + j] = val % 2 == 1;
+            val >>= 1;
+        }
+    }
+
+    let iv_string = "0D74DB42A91077DE45AC000000000000".to_string();
+    let mut iv = [false; 128];
+
+    for i in (0..iv_string.len()).step_by(2) {
+        let mut val: u8 = u8::from_str_radix(&iv_string[i..i + 2], 16).unwrap();
+        for j in 0..8 {
+            iv[8 * (i >> 1) + j] = val % 2 == 1;
+            val >>= 1;
+        }
+    }
```

Comment:
Seems like this piece of code is often copy pasted, could it be moved in a function ?

Reply:
I agree ! It's only used in tests and benches though, I'm open to suggestions to where I should put it (hoping it doesn't get compiled into the prod binary). Like a `utils.rs` file in a test folder ?

Reply:
We could do something like in the tfhe benches https://github.com/zama-ai/tfhe-rs/blob/7ea13715eeaf006cf39849b4fd22226d2deb6b1a/tfhe/benches/integer/bench.rs#L3C1-L3C28

But we can probably leave with how the current code is